resources/css/bem/changelog-md.less

```diff
@@ -26,6 +26,14 @@
     }
   }
 
+  &__link {
```

Comment:
just removing the `a` styling is enough (or more like it's overriding this styling and this change doesn't do anything)

also
> Since changelogs now use the `changelog-entry` CSS-class that already fixed this issue a long time ago, I wanted to fix this problem for previous posts that used the `changelog-md` CSS-class and match their colors for better readability.

they're for different part of the changelog and still used.

Reply:
Removed the a-styling and the part of `&__link` that would override the a-styling. Hover-part is staying to correctly match colors with `changelog-entry`. Hope that's fine now!